redaxo/src/addons/structure/plugins/content/lib/api_template.php

```diff
@@ -107,10 +90,12 @@ public static function getFilePath($template_id)
             return false;
         }
 
-        return self::getTemplatesDir() . '/' . $template_id . '.template';
+        return rex_template_cache::getPath($template_id);
     }
 
     /**
+     * @deprecated since 2.11, use `rex_template_cache` instead
```

Comment:
```suggestion
     * @deprecated since 5.11, use `rex_template_cache` instead
```

Reply:
Es ist die Addonversion. Ich war mir auch unsicher, aber so ist es eigentlich korrekter, oder?
Es ist ja auch deprecated, wenn jemand structure 2.11 in R5.10 nutzt (weiß gerade nicht, ob wir das zulassen).
Andererseits ist die Core-Version natürlich geläufiger.

Reply:
wenn es die addon version ist, sollte es zumindest textuell eindeutig erkennbar sein.. passt dann auch für mich